pkg/termination/termination.go

```
@@ -58,6 +65,47 @@ func NewHandler(logger logr.Logger, cfg *rest.Config, pollInterval time.Duration
+	req, err := http.NewRequest("GET", pollURL.String(), nil)
+	if err != nil {
+		return false
+	}
+
+	req.Header.Add("Metadata", "true")
+
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return false
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)